flightlib/include/flightlib/bridges/unity_message_types.hpp

```diff
@@ -38,6 +38,9 @@ struct Camera_t {
   int width{1024};
   int height{768};
   Scalar fov{70.0f};
+  // Clip Planes for the different layers
+  std::vector<Scalar> nearClipPlane{0.01, 0.01, 0.01, 0.01};
+  std::vector<Scalar> farClipPlane{1000.0, 100.0, 1000.0, 1000.0};
```

Comment:
please follow the variable name convention in C++ `neal_clip_plane`, `far_clip_plane`